### PR TITLE
[TIMOB-23303] iOS: Today extension doesn't show any content

### PR DESCRIPTION
**Pull request by cheekiatng**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23303

### CODE REVIEW

**Comment by cheekiatng** [comment]
DO NOT MERGE. Error in subsequent builds.

File: iphone/cli/commands/_build.js [hunk 1]
```diff
@@ -2916,6 +2920,48 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 					comment: xobjs.PBXNativeTarget[targetUuid].productReference_comment
 				});
 
+				// find the extension frameworks and resources group
+				for (var key in extObjs.PBXGroup) {
```
Comment (hansemannn) [inline iphone/cli/commands/_build.js]: Would it make sense to use `Object.keys(extObjs.PBXGroup).forEach(function (key) {} ` here? Just a minor styling thing. 😊

Comment (cb1kenobi) [inline iphone/cli/commands/_build.js]: Agreed, use `Object.keys()`.

File: iphone/cli/commands/_build.js [hunk 2]
```diff
@@ -2916,6 +2920,48 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
...
+					if(extObjs.PBXGroup[key] === 'Frameworks') {
+						extFrameworksGroup = key.split('_')[0];
+					}
+					if(extObjs.PBXGroup[key] === 'Resources') {
+						extResourcesGroup = key.split('_')[0];
+					}
+				}
+
+				// add the extension frameworks to the frameworks group
+				if(extFrameworksGroup) {
```
Comment (cb1kenobi) [inline iphone/cli/commands/_build.js]: Missing space between `if` and `(`. Please fix all instances in this commit.

**Comment by cheekiatng** [comment]
@cb1kenobi comments addressed.

**Comment by hansemannn** [comment]
@cheekiatng Can you pull from master? Ash fixed a travis issue that needs to be included to pass validation.

**Comment by cheekiatng** [comment]
@hansemannn tis is ready

**Comment by hansemannn** [comment]
Approved!

**Pull request merged**